index.js

```diff
@@ -1,4 +1,8 @@
 /** @format */
+// Disable the yellow box, this is only containing warnings and will kill the automation
+// The yellow box is also shown during automation on the dev build when the automation goes to fast
+// for React Native
+console.disableYellowBox = true;
```

Comment:
Does it make sense to add tasks to re-enable this on a regular basis so that we can see and eliminate the warnings? Or do you think that we can just completely ignore them? I was seeing warnings about events being sent to nonexistent handlers, so I wonder if it would make sense to clean those up

Reply:
Well, based on my experience we can ignore them, only when we do a upgrade to a new version of RN we should enable it for a while